### PR TITLE
Add the mention of supported event actions

### PR DESCRIPTION
**Pull request by hellcp-work**
Taken from https://github.com/openSUSE/open-build-service/blob/729c0721e9903f1824aa8e33dedd5396c6ba0a9c/src/api/app/models/scm_webhook.rb#L10-L11

### CODE REVIEW

File: xml/obs_scm_ci_workflow_integration.xml [hunk 1]
```diff
@@ -199,6 +199,58 @@
           </listitem>
         </itemizedlist>
 
+        <para>In addition, the Pull request events contain an field called
```
Comment (danidoni) [inline xml/obs_scm_ci_workflow_integration.xml:202]:
```suggestion
        <para>In addition, the Pull request events contains a field called
```

File: xml/obs_scm_ci_workflow_integration.xml [hunk 2]
```diff
@@ -199,6 +199,58 @@
...
+          <listitem>
+            <para>synchronize</para>
+          </listitem>
+
+          <listitem>
+            <para>synchronized</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For Gitlab's Merge request event, the following actions are
```
Comment (danidoni) [inline xml/obs_scm_ci_workflow_integration.xml:229]:
```suggestion
        <para>For Gitlab's Merge request events, the following actions are
```
To be coherent with the previous paragraph

**Review by danidoni — APPROVED** [review]

**Review by saraycp — CHANGES_REQUESTED** [review]

File: xml/obs_scm_ci_workflow_integration.xml [hunk 3]
```diff
@@ -199,6 +199,58 @@
...
+          <listitem>
+            <para>reopen</para>
+          </listitem>
+
+          <listitem>
+            <para>update</para>
+          </listitem>
+        </itemizedlist>
+
         <para>Other SCMs like GitLab have similar behaviour; the main
```
Comment (saraycp) [inline xml/obs_scm_ci_workflow_integration.xml:254]: The paragraph which starts in this line should be moved right after the events list. 
Take into account we only list GitHub events and use this paragraph to invite users to discover the events from other SCMs.

Unlike the actions, that we list all of them (GitHub, GitLab, Gitea).

Comment (saraycp) [reply xml/obs_scm_ci_workflow_integration.xml:254]: An alternative would be listing all the possible events for all the SCMs. And then rephrase that paragraph to simply invite the users to read more (and don't move it).

**Review by saraycp — CHANGES_REQUESTED** [review]

**Pull request merged**